gdal/ogr/ogrsf_frmts/ngw/gdalngwdataset.cpp

```diff
@@ -312,6 +316,7 @@ bool OGRNGWDataset::Init(int nOpenFlagsIn)
 
                 // Create raster dataset.
                 std::string osRasterUrl = NGWAPI::GetTMS(osUrl, osResourceId);
+                char* pszRasterUrl = CPLEscapeString(osRasterUrl.c_str(), -1, CPLES_XML);
```

Comment:
must be CPLFree() 'ed

Reply:
Fixed per 12da9726242ce149b5e0abb5614ea2ec363fe952